features.yaml

```diff
@@ -827,6 +827,12 @@ groups:
       help: MD5 hardware acceleration present.
     - name: periph_hash_sha_1
       help: SHA-1 hardware acceleration present.
+    - name: periph_hash_sha3_256
+      help: SHA-3/256 hardware acceleration present.
+    - name: periph_hash_sha3_384
+      help: SHA-3/384 hardware acceleration present.
+    - name: periph_hash_sha3_512
+      help: SHA-3/512 hardware acceleration present.
```

Comment:
Would you mind moving them below all the SHA-2 functions, here and in the other files? Just for consistency reasons.